src/script/ptl-tool.py

```diff
@@ -231,16 +231,8 @@ def build_branch(args):
     if branch == 'HEAD':
         log.info("Leaving HEAD detached; no branch anchors your commits")
     else:
-        # Delete test branch if it already existed
-        try:
-            getattr(G.branches, branch).delete(
-                    G, getattr(G.branches, branch), force=True)
-            log.info("Deleted old test branch %s" % branch)
-        except AttributeError:
-            pass
-
-        G.create_head(branch)
-        log.info("Created branch {branch}".format(branch=branch))
+        G.head.reference = G.create_head(branch, force=True)
+        log.info("Checked out new branch {branch}".format(branch=branch))
```

Comment:
I think this change needs to be combined. I mean, if a test branch already exist, then use it. Don't create a new one. If it doesn't exist then create a new one.

Because when I test it, I saw that if I execute the below commands, to merge 18107 and 10108 separately,
```
$ src/script/ptl-tool.py --base master --label wip-jcollin-testing 18107
$ src/script/ptl-tool.py --base master --label wip-jcollin-testing 18108
```
The changes provided by 18107 gets overwritten by the second execution of ptl-tool.py !

Reply:
That's intended. You should use `--base-path heads/ --base wip-jcollin-testing --branch wip-jcollin-testing` instead.

Reply:
It gives me:
```
Detaching HEAD onto base: wip-jcollin-testing-20171007
Branch wip-jcollin-testing-20171007 does not exist!
```

Reply:
Sorry, use `--base-path refs/heads/`.